Taskcpp.cpp

```diff
@@ -0,0 +1,116 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <numeric>
+
+using namespace std;
+
+
+enum Profession {
+	Unknown,
+	Developer,
+	Manager,
+	QA,
+	Marketing,
+	HR,
+};
+constexpr unsigned int str2int(const char* str, int h = 0)
+{
+	return !str[h] ? 5381 : (str2int(str, h + 1) * 33) ^ str[h];
+}
+Profession professionchecker(string s) {
+
+	switch (str2int(s.c_str())) {
+	case  str2int("Developer"):
+		return Profession::Developer;
+		break;
+	case  str2int("Manager"):
+		return Profession::Manager;
+		break;
+	case  str2int("QA"):
+		return Profession::QA;
+		break;
+	case  str2int("Marketing"):
+		return Profession::Marketing;
+		break;
+	case  str2int("HR"):
+		return Profession::HR;
+		break;
+	case  str2int("Unknown"):
+		return Profession::Unknown;
+		break;
+	}
+};
+class  Employee {
+public:
+	string name;
+	string surname;
+	int age;
+	Profession profession;
+	Employee(string empl_name, string empl_surname, int empl_age, string empl_profession) {
+		name = empl_name;
+		surname = empl_surname;
+		age = empl_age;
+		profession = professionchecker
+		(empl_profession);
+	}
+};
+void EmployeeChecker(int n, vector<Employee*> employees, string namefilter, string surnamefilter, int agefilter, string professionfilter) {
```

Comment:
Make it work in this way
Input filter name(- if any): -
Input filter surname(- if any): Asatryan
Input age filter(- if any): 14
Input age filtering method(<, >, <=, >=, ==): >
Input proffession(- if any): -